corehq/apps/users/views/__init__.py

```diff
@@ -572,7 +572,7 @@ def page_context(self):
             'can_edit_roles': self.can_edit_roles,
             'default_role': UserRole.get_default(),
             'report_list': get_possible_reports(self.domain),
-            'web_apps_list': get_cloudcare_apps(self.domain),
+            #'web_apps_list': get_cloudcare_apps(self.domain),
```

Comment:
Why not just delete this? I don't see it being removed in https://github.com/dimagi/commcare-hq/pull/28576/ either.

Reply:
And doesn't this need to pass `allowed_apps_list` instead?

Reply:
Sorry, this should not be here.
We don't need `allowed_apps_list` here because we already have `app_list` which is giving us all the applications which we actually need. We would be checking the ones which are in `allowed_app_list` permissions on UI.